### ISSUE TITLE
API reference doesn't show members for .Net Standard types not referenced from NETStandard.Library

### ISSUE THREAD

**Issue by svick** [dotnet/docs#1849]
For types that have .Net Standard packages, but are not part of `NETStandard.Library`, it seems the new API reference behaves as if those types did not have .Net Standard versions at all (i.e. it hides all their members).

For example, this happens for [`XmlNode`](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmlnode?view=netstandard-1.6) (the URL says `netstandard-1.6`, but it seems it changes to `netframework-4.7` on its own, so you have to use the dropdown to see it), which exists in the `System.Xml.XmlDocument` package, which has a version for .Net Standard 1.3.

I think for such types, their .Net Standard version should be shown.

https://apisof.net/ has a very similar issue (https://github.com/terrajobst/apisof.net/issues/2), which could be related.

**Comment by mairaw** [comment]
@dend can you look at this? It seems that selecting .NET standard is not sticky.

**Comment by mairaw** [comment]
@svick I know this is way older than #3132, but they seem to be regarding the same issue. Is it ok to close this and track that issue on that one?

**Comment by svick** [comment]
@mairaw Of course. If it was up to me, I would probably choose the older issue or the more general one and this is both. But it's not. 🙂

**Comment by mairaw** [comment]
Yes, that's usually my approach. It's just we got the attention of the dev team on that one, so it will help move things. I'll make that one more generic. Thanks for your understanding.

**Comment by mairaw** [comment]
Reopening this one. The WCF one will be handled differently.
